x-pack/platform/test/functional/page_objects/lens_page.ts

```diff
@@ -1713,9 +1723,18 @@ export function LensPageProvider({ getService, getPageObjects }: FtrProviderCont
       for (let i = 0; i < tabs.length; i++) {
         const tabText = await tabs[i].getVisibleText();
         if (tabText === name) {
-          await tabs[i].click(); // Click to make it active
+          // Check if the tab is already active
+          const isActive = await tabs[i].getAttribute('aria-selected');
+          if (isActive === 'true') {
+            await testSubjects.exists(`lns-layerPanel-${i}`);
+            return;
+          }
+
+          // Click to make it active
+          await tabs[i].click();
+
           // Wait for the layer panel to render
-          await retry.waitFor('layer panel to be visible', async () => {
+          await retry.waitForWithTimeout('layer panel to be visible', 1000, async () => {
             return await testSubjects.exists(`lns-layerPanel-${i}`, { timeout: 1000 });
           });
```

Comment:
I wonder if the `timeout` is needed on the `testSubjects.exists` when wrapping with `waitForWithTimeout`. Not sure 🤔

Reply:
And to be clear: `testSubjects.exists` always runs with its internal retry mechanism - if you don't provide a `timeout` it'll run with the default value. So the right fix here would not be to remove the `timeout` parameter from `testSubjects.exists` but to remove the wrapping `waitForWithTimeout`.